airbyte-integrations/bases/standard-destination-test/src/main/java/io/airbyte/integrations/standardtest/destination/TestDestination.java

```diff
@@ -240,7 +245,7 @@ public void testSync(String messagesFilename, String catalogFilename) throws Exc
         .map(record -> Jsons.deserialize(record, AirbyteMessage.class)).collect(Collectors.toList());
     runSync(getConfig(), messages, catalog);
 
-    assertSameMessages(messages, retrieveRecords(testEnv, catalog.getStreams().get(0).getName()));
+    assertSameMessages(messages, retrieveRecordsForCatalog(catalog));
```

Comment:
we now have to handle test cases where the input data has more than one stream. all of the changes in this class are to handle that.